numpy-tutorial/curve_grades.py

```diff
@@ -0,0 +1,33 @@
+"""Introduction example: curving a series of test grades using Numpy."""
+
+import numpy as np
+
+
```

Comment:
🎩 **Style Tip**

Use consistent code formatting. Some of your files follow PEP 8 with respect to the number of blank lines, while others don't, for example.